tests/unit/Storefront/Theme/ThemeScriptsTest.php

```diff
@@ -119,6 +131,12 @@ public function testInheritsFromBase(): void
         $request->attributes->set(SalesChannelRequest::ATTRIBUTE_THEME_ID, 'Storefront');
         $request->attributes->set(SalesChannelRequest::ATTRIBUTE_THEME_NAME, null);
         $request->attributes->set(SalesChannelRequest::ATTRIBUTE_THEME_BASE_NAME, 'Storefront');
+        $request->attributes->set(PlatformRequest::ATTRIBUTE_CONTEXT_OBJECT, Context::createCLIContext());
+
+        $salesChannelContext = $this->createMock(SalesChannelContext::class);
+        $salesChannelContext->method('getContext')->willReturn(Context::createCLIContext());
+        $request->attributes->set(PlatformRequest::ATTRIBUTE_SALES_CHANNEL_CONTEXT_OBJECT, $salesChannelContext);
```

Comment:
yes, the test is running again, with all those additions, but it is not really ensured, that the described problem is fixed

Reply:
Yes it is. I change attributes of "testInheritsFromBase", theme child can't it is parent itself

Reply:
looks better now :+1: